docs/openapi/schemas-pn-notification-v1.yaml

```diff
@@ -485,3 +485,11 @@ components:
           description: >-
             Stima del numero di secondi da aspettare prima che il contenuto del 
             documento sia disponibile per il download.
+
+            
+    NotificationQuickAccessLinkTokens:
```

Comment:
Di solito ci siamo sempre restituiti oggetti. Penso non sia un problema in fase di generazione del client lato pn-delivery-push

Reply:
non è problema.